### PR TITLE
Fanfiction Presence and Duelingnexus Update

### PR DESCRIPTION
**Pull request by sunx2**
+ Added fanfiction presence.

+ Added timestamps in duelingnexus.

### CODE REVIEW

File: Fanfiction/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "name": "Nekolen",
+        "id": "270125856679002113"
+    },
+    "service": "Fanfiction",
+    "description": {
+        "en": "FanFiction | unleash your imagination"
+    },
+    "url": "www.fanfiction.net",
+    "version": "1.0",
+    "logo": "https://i1.wp.com/geezerwench.com/wp-content/uploads/2017/10/Fanfiction-net-logo-400x400.png?fit=400%2C400",
```
Comment (frozaum) [inline Fanfiction/dist/metadata.json]: low quality logo

File: Fanfiction/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "thumbnail": "https://images-eu.ssl-images-amazon.com/images/I/417sQ2crmUL.jpg",
```
Comment (frozaum) [inline Fanfiction/dist/metadata.json]: can you change this to a screenshot of the website?

File: Fanfiction/dist/metadata.json (hunk 1 above)
Comment (sunx2) [reply Fanfiction/dist/metadata.json]: i had to magnify the logo because it wasnt fitting in 512*512

File: Fanfiction/dist/metadata.json (hunk 2 above)
Comment (sunx2) [reply Fanfiction/dist/metadata.json]: that website looks garbo

Comment (frozaum) [reply Fanfiction/dist/metadata.json]: boi, at least put a screenshot like the other presences does

Comment (sunx2) [reply Fanfiction/dist/metadata.json]: haha ok dad

File: Fanfiction/dist/metadata.json (hunk 1 above)
Comment (frozaum) [reply Fanfiction/dist/metadata.json]: still, it has a low resolution

and you need to change the version number on dueling nexus

**Comment by sunx2** [comment]
Changed the duelingnexus version on metadata

**Comment by sunx2** [comment]
Fixed Logo and version number

File: Fanfiction/dist/metadata.json (hunk 1 above)
Comment (sunx2) [reply Fanfiction/dist/metadata.json]: i changed the version number and placed logo and thumbnail properly

File: Fanfiction/dist/metadata.json [hunk 3]
```diff
@@ -9,8 +9,9 @@
     },
     "url": "www.fanfiction.net",
     "version": "1.0",
-    "logo": "https://i1.wp.com/geezerwench.com/wp-content/uploads/2017/10/Fanfiction-net-logo-400x400.png?fit=400%2C400",
-    "thumbnail": "https://images-eu.ssl-images-amazon.com/images/I/417sQ2crmUL.jpg",
+    
+    "logo": "https://images-eu.ssl-images-amazon.com/images/I/417sQ2crmUL.jpg",
```
Comment (Bas950) [inline Fanfiction/dist/metadata.json]: Logo needs to be a square image and thumbnail a preview of the website.

**Comment by sunx2** [comment]
idk why fanfiction looks so ugly and still popular >.>

**Pull request merged**